src/BasePlatform.ts

```diff
@@ -365,10 +368,6 @@ export default abstract class BasePlatform {
         window.location.href = mxClient.getSsoLoginUrl(callbackUrl.toString(), loginType, idpId); // redirect to SSO
     }
 
-    onKeyDown(ev: KeyboardEvent): boolean {
```

Comment:
Did you delete this method by intent?

Reply:
It doesn't seem to be used anywhere - yes, it was intentional